pkg/server/desthost_backend_manager.go

```diff
@@ -0,0 +1,39 @@
+package server
```

Comment:
Needs a 2020 Kubernetes Authors copyright block

Reply:
This should have caused a header check failure. If not please let me know.

Reply:
Will a wrong copyright year cause a header check failure?

Reply:
+1